Regrab any previous gui focus after closing console.

Wierdox opened the pull request:
For me at least this is important, I have logic that assumes there is always a focused UI element in some menus. I imagine in most cases this is fine to do, but maybe it should be a toggleable option?

Also just to let you know, I have another PR to add animated console opening/closing, but I'll wait for my PRs to be OKed or not first since it would touch some of the same code.

limbonaut commented on limbo_console.gd:
```diff
@@ -126,6 +128,8 @@ func hide_console() -> void:
 	if _control.visible:
 		_control.hide()
 		get_tree().paused = false
+		if _previous_gui_focus:
```
Need to use `is_instace_valid()` because a control may be removed as an effect of a command

limbonaut commented:
> For me at least this is important, I have logic that assumes there is always a focused UI element in some menus. I imagine in most cases this is fine to do, but maybe it should be a toggleable option?

I agree, this is fine. No need for an option until proven problematic.

Wierdox commented:
Okay, added your suggestions and I also rebased this off of #4, so if you merge them in order it should be clean. Or so I think, I'm not well versed in that area.

limbonaut reviewed (approved):
Looks good. GitHub's **Files Changed** panel is confused, showing previous PR changes as well. But I think it'll merge just fine.
Thanks.
